### PR TITLE
Bruce jwttsso more fats

### PR DESCRIPTION
**Pull request by brutif**

### CODE REVIEW

**Comment by brutif** [comment]
#build

**Comment by brutif** [comment]
#build

**Comment by brutif** [comment]
#build

**Comment by brutif** [comment]
#build

File: dev/com.ibm.ws.security.jwtsso_fat/fat/src/com/ibm/ws/security/jwtsso/fat/CookieProcessingTests.java [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+    @Mode(TestMode.LITE)
+    @Test
+    public void test_CookieReplay() throws Exception {
+        reconfigServer("server.xml");
+        doHappyPath();
+        String responseStr = response.getWebResponse().getContentAsString();
+        String beginStr = "cookie: jwtToken value: ";
+        int begin = responseStr.indexOf(beginStr) + beginStr.length();
+        int end = responseStr.indexOf("\n", begin);
+        String token = responseStr.substring(begin, end);
```
Comment (ayoho) [inline dev/com.ibm.ws.security.jwtsso_fat/fat/src/com/ibm/ws/security/jwtsso/fat/CookieProcessingTests.java]: As I mentioned the other day, you should be able to do `wc.getCookieManager().getCookie("jwtCookie")` to get the actual cookie object. You shouldn't have to parse the servlet output to get the cookie value.

**Pull request merged**